content/zh-cn/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -36,28 +35,26 @@ and environment. [Options for Highly Available topology](/docs/setup/production-
 If you encounter issues with setting up the HA cluster, please report these
 in the kubeadm [issue tracker](https://github.com/kubernetes/kubeadm/issues/new).
 
-See also the [upgrade documentation](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15).
+See also the [upgrade documentation](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/).
 -->
-在下一步之前，你应该仔细考虑哪种方法更好的满足你的应用程序和环境的需求。 
+在下一步之前，你应该仔细考虑哪种方法更好地满足你的应用程序和环境的需求。
 [高可用拓扑选项](/zh-cn/docs/setup/production-environment/tools/kubeadm/ha-topology/) 讲述了每种方法的优缺点。
 
 如果你在安装 HA 集群时遇到问题，请在 kubeadm [问题跟踪](https://github.com/kubernetes/kubeadm/issues/new)里向我们提供反馈。
 
-你也可以阅读[升级文档](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
+你也可以阅读[升级文档](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)。
 <!--
 This page does not address running your cluster on a cloud provider. In a cloud
 environment, neither approach documented here works with Service objects of type
 LoadBalancer, or with dynamic PersistentVolumes.
 -->
 {{< caution >}}
```

Comment:
In newer translations, we use this format. I think you already know it, just missed in this resync PR.

```
{{< caution >}}
<!--
English text.
-->
中文翻譯
{{< /caution >}}
```

Reply:
ok, have changed some paras like the issue in this topic.